src/OpenSpaceToolkit/Astrodynamics/Trajectory/Sequence.cpp

```diff
@@ -93,6 +97,19 @@ Mass Sequence::Solution::computeDeltaMass() const
     return Mass::Kilograms(getInitialMass().inKilograms() - getFinalMass().inKilograms());
 }
 
+Array<Maneuver> Sequence::Solution::extractManeuvers(const Shared<const Frame>& aFrameSPtr) const
+{
+    Array<Maneuver> maneuvers = Array<Maneuver>::Empty();
+
+    for (const Segment::Solution& segmentSolution : this->segmentSolutions)
+    {
+        const Array<Maneuver> segmentManeuvers = segmentSolution.extractManeuvers(aFrameSPtr);
+        maneuvers.add(segmentManeuvers);
```

Comment:
just make sure you end up with an `Array<Maneuver>` and not a `Array<Array<Maneuver>>`